### PR TITLE
Add support for extra processing defined in _00build.conf

### PR DESCRIPTION
**Pull request by peabee**
e.g. for devx manipulation

### CODE REVIEW

File: woof-code/3builddistro [hunk 1]
```diff
@@ -332,6 +332,9 @@ if [ -n "$PETBUILDS" ]; then
 		echo "Building without qemu-user-static"
 	fi
 
+	# set EXTRA_PROCESSING in _00build.conf to perform any processing required - e.g. for devx
+	[ $EXTRA_PROCESSING ] && . $EXTRA_PROCESSING
```
Comment (01micko) [inline woof-code/3builddistro]: This test _can_ (not saying it is) be ambiguous without a string comparison operator, `-n` for non-zero string length, `-z` for zero length.

**Review by 01micko — CHANGES_REQUESTED** [review]

File: woof-code/3builddistro (hunk 1 above)
Comment (dimkr) [reply woof-code/3builddistro]: And there's a syntax error if it's undefined, you need to wrap `$EXTRA_PROCESSING` with quotes

File: woof-code/3builddistro [hunk 2]
```diff
@@ -332,6 +332,9 @@ if [ -n "$PETBUILDS" ]; then
 		echo "Building without qemu-user-static"
 	fi
 
+	# set EXTRA_PROCESSING in _00build.conf to perform any processing required - e.g. for devx
```
Comment (dimkr) [inline woof-code/3builddistro]: Why is this manipulation of devx done only if PETBUILDS is non-empty? I'd do it earlier, at line 306, https://github.com/puppylinux-woof-CE/woof-CE/blob/d5b9484598f2cfa28328ce841eb4d8be6e1d43c2/woof-code/3builddistro#L306

**Comment by peabee** [comment]
devx was only an e.g. so requested changes done but at line 312.

**Review by 01micko — APPROVED** [review]

**Pull request merged**